powershell/CompiledHelpers/VstsTaskSdk.csproj

```diff
@@ -11,9 +11,10 @@
     <AssemblyName>VstsTaskSdk</AssemblyName>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>FinalPublicKey.snk</AssemblyOriginatorKeyFile>
-    <DelaySign>true</DelaySign>    
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <DelaySign>true</DelaySign>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
Can this be the same as for an agent?
https://github.com/microsoft/azure-pipelines-agent/blob/4288c014a39b8968228a3f2f85e3da912ae3da79/src/Agent.Service/Windows/AgentService.csproj#L15

Reply:
The hosted agent which runs this pipeline doesn't contain old msbuild, thus I upgraded it. 
I will revert it back to 4.0 and add old msbuild by task on pipeline.